parts/k8s/kubernetesmastercustomdata.yml

```diff
@@ -292,17 +292,15 @@ MASTER_ARTIFACTS_CONFIG_PLACEHOLDER
   content: |
     #!/bin/bash
     set -x
-    retrycmd_if_failure() { for i in $(seq 1 36); do $@; [ $? -eq 0  ] && break || sleep 5; done ; }
+    retrycmd_if_failure() { retries=$1; shift; for i in $(seq 1 $retries); do ${@}; [ $? -eq 0  ] && break || sleep 5; done ; }
     ETCD_VER=v{{WrapAsVariable "etcdVersion"}}
     DOWNLOAD_URL={{WrapAsVariable "etcdDownloadURLBase"}}
-    mkdir -p /tmp/etcd-download
-    retrycmd_if_failure curl --retry 5 --retry-delay 10 --retry-max-time 30 -L ${DOWNLOAD_URL}/etcd-${ETCD_VER}-linux-amd64.tar.gz -o /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+    retrycmd_if_failure 5 curl --retry 5 --retry-delay 10 --retry-max-time 30 -L ${DOWNLOAD_URL}/etcd-${ETCD_VER}-linux-amd64.tar.gz -o /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
     tar xzvf /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz -C /usr/bin/ --strip-components=1
     useradd -U "etcd" > /dev/null 2>&1
-    usermod -p "$(head -c 32 /dev/urandom | base64)" "etcd"
-    passwd -u "etcd" > /dev/null
-    systemctl daemon-reload
-    systemctl enable etcd.service
```

Comment:
I removed this line because it wasn't doing anything and just returning an error, any objections?

Reply:
Removing erratic no ops is 👍